src/interactable_booster.c

```diff
@@ -0,0 +1,160 @@
+#include "global.h"
+#include "malloc_vram.h"
+#include "m4a.h"
+
+#include "game.h"
+#include "task.h"
+
+#include "interactable.h"
+#include "sprite.h"
+
+#include "constants/move_states.h"
+#include "constants/songs.h"
+
+typedef struct {
+    /* 0x00 */ SpriteBase base;
+    /* 0x0C */ struct UNK_0808B3FC_UNK240 main;
+} Sprite_Booster;
+
+extern void Task_Interactable_Booster(void);
+extern void TaskDestructor_80095E8(struct Task *);
+
+extern bool32 sub_800C204(struct UNK_0808B3FC_UNK240 *, s16, s16, s16,
+                          struct SomeStruct_59E0 *, u32);
+extern void sub_80218E4(struct SomeStruct_59E0 *);
+extern void sub_8023B5C(struct SomeStruct_59E0 *, s8);
+
+extern const struct UNK_080E0D64 gUnknown_080D942C[2][6];
+
+#define BOOSTER_SPEED 3072
```

Comment:
Cool we can now set this 😎

Reply:
Not completely freely while the whole game doesn't match, but yeah! :D